pkg/server/services/lease.go

```
@@ -0,0 +1,113 @@
+	// only update action
+	switch eventType.Action {
+	case types.UpdateRequestAction:
+		_, err := l.client.CoordinationV1().Leases(lease.Namespace).Update(ctx, lease, metav1.UpdateOptions{})
+		return err
+	default:
+		return fmt.Errorf("unsupported lease action %s", eventType.Action)
+	}
+}
+
+func (l LeaseService) RegisterHandler(handler server.EventHandler) {
+	l.informer.Informer().AddEventHandler(&cache.ResourceEventHandlerFuncs{
```

golangci-lint (1.64.8)

Error return value of `(k8s.io/client-go/tools/cache.SharedInformer).AddEventHandler` is not checked

(errcheck)